packages/bigtable/src/mutation.js

```diff
@@ -80,6 +80,10 @@ Mutation.convertFromBytes = function(bytes) {
  * @return {buffer}
  */
 Mutation.convertToBytes = function(data) {
+  if (data instanceof Buffer) {
```

Comment:
`Buffer.isBuffer()` does not work in 0.12.x (pretty sure at least). But we can definitely use that module if you'd prefer that to `instanceof`.